apps/condo/domains/user/integration/identity/SberIdIdentityIntegration.js

```diff
@@ -0,0 +1,187 @@
+const { customAlphabet } = require('nanoid')
+const { isNil } = require('lodash')
+const jwtDecode = require('jwt-decode')
+const axios = require('axios').default
+const https = require('https')
+const { generators } = require('openid-client')
+
+const conf = require('@open-condo/config')
+
+const { SBER_ID_IDP_TYPE } = require('@condo/domains/user/constants/common')
+const { USER_EXTERNAL_IDENTITY_CALLBACK_PATH } = require('@condo/domains/user/constants/links')
+const AbstractIdentityIntegration = require('@condo/domains/user/integration/identity/AbstractIdentityIntegration')
+
+// get sber id configuration params
+const SBER_ID_CONFIG = process.env.SBER_ID_CONFIG ? JSON.parse(process.env.SBER_ID_CONFIG) : {}
+const {
+    authorizeUrl,
+    tokenUrl,
+    userInfoUrl,
+    scope,
+    clientId,
+    clientSecret,
+    redirectUri,
+    cert,
+    key,
+    verifyServerSsl,
+} = SBER_ID_CONFIG
+const callbackPath = USER_EXTERNAL_IDENTITY_CALLBACK_PATH.replace(':type', SBER_ID_IDP_TYPE)
+const callbackUri = redirectUri || `${conf.SERVER_URL}${callbackPath}`
+const axiosTimeout = 10000
+
+// instantiate httpsAgent in order to support mTLS communication with sber id servers
+const httpsAgent = new https.Agent({
+    rejectUnauthorized: verifyServerSsl,
+    cert,
+    key,
+})
+
+// instantiate request id generator
+const nanoid = customAlphabet('1234567890abcdef', 32)
+
+class SberIdIdentityIntegration extends AbstractIdentityIntegration {
```

Comment:
why we don't have the same class for SBBOL ?

Reply:
Identity Integration provides an interface to communicate with external system that going to auth user in our platform. SBBOL not a system that we use for user auth